backend/codesirius/problems/urls.py

```
@@ -17,17 +17,19 @@
         name="problem-retrieve-update-destroy",
     ),
-    path("tags", TagListCreateAPIView.as_view(), name="tag-list-create"),
+    path("tags/", TagListCreateAPIView.as_view(), name="tag-list-create"),
     path(
         "tags/<int:pk>",
         TagRetrieveUpdateDestroyAPIView.as_view(),
         name="tag-retrieve-update-destroy",
     ),
-    path("languages", LanguageListCreateAPIView.as_view(), name="language-list-create"),
+    path(
+        "languages/", LanguageListCreateAPIView.as_view(), name="language-list-create"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)